### ISSUE TITLE
PRIORITY: Looking for Open Source Volunteers in What's App India, etc.

### ISSUE THREAD

**Issue by sadrev25** [Morningstar88/X#13]
### Volunteer Coders

Hi everyone,

Does anyone have any JS/CSS skills?

We have an open source project... Kalki. Here's the url..

Our aim is to make a simple browser app that can help people raise their first 500USD. 

We have a few tasks here, if you want to help.

We're volunteers, so are not hiring right now, but you can help us open source, and get some experience and training.

Please come to our issues page and talk.

## Marketers

Hi everyone… are there any marketers who want to help us try and raise money for Kali.

**Comment by sadrev25** [comment]
Hi, jake, What my idea is, Now we are concentrating on our Kalki promotion and fundraising Kalki, and it's going well. but we need to create & develop our Kalki, as I told good products only make a huge. success. so we need to fix problems in Kalki.
1. we need to build an analyst team to analyze each startup.
2. import anlalysed data in our kalki.
3. our Kalki linkage between stocks and customer right, so our customers know about the summary of every start-up like(USP, value of startup, tenure, lifetime)
above idea improves more efficiency jake
The estimated budget for this project is 63 USD. I will attach a photo of the stock market app, whose following is like this. since our promotion team of WhatsApp, Instagram, and Facebook are ready, just we need to make our model perfect.
NEXT IDEA
As you told me, they are many freelancers whose offer to fundraise and help to get free volunteers. this will help to find more volunteers since we did not get volunteers. those freelancers are charging different upon the tasks. I shortlisted four guys,
1. Vivek prankish-he is guaranteed upto 30 free volunteers, charges 30usd
2. Sharmila-she is a member of a social organizing group in India, she has more contact,she guarantees above 50 volunteers, charges 40usd
3. Franck-guaranteed 20, charges 30 USD
4. gowtham-he is working in a media company in India, and he will charge free volunteers, one free volunteer-2usd
after this project &hire, we seem to have small company jake.
will have workers and an analyst team.After this we can claim donations and funds easily.  let's hope for the best.
CHEERS!!!

**Comment by Morningstar88** [comment]
Hi!

For the first idea, I don't have the spare budget right now...

*
But the second one , raising money, is interesting... and the prices are good. 

I'll get back to you tomorrow. Let's raise money quickly.

Glad you found some good people at fair prices. 
Peace!


-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 8, 2023, 20:27 by ***@***.***:

**Comment by sadrev25** [comment]
Sure jake, will meet on Tuesday.

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>Cool. We can go to Subway again. 

We need to research where Lions International is, and prepare a pitch for them.

**Comment by sadrev25** [comment]
Sure ask mike,he will give some suggestions

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>PS, Mukesh,

What do you think of Goa?

I need to be near the sea, and love nuts. 

Maybe a good place to start in India?

**Comment by sadrev25** [comment]
Yeah nice place,pne of the happenings place in india.Lets meet tomorrow

**Comment by Morningstar88** [comment]
I need to be near a beach to be 100% healthy. So either Goa or Chennai.
Can we meet at 5 or 6pm near my house? 6pm is best, but 5pm if you prefer earlier. please  let me know. 

Then go eat in Subway.

Please bring your laptop, if possible. 
*

I have no spare money tomorrow for advertising or other projects, but the website should be better.

Maybe we can go to Lions on Thursday. 

Need to raise donations soon, for both of us. 

Once we get some donations, can advertise, then look for volunteers. 

if your friends can guarantee javascript or fundraising volunteers, I can consider it, but funds are low right now.

Let's see. 

Peace.

**Comment by sadrev25** [comment]
Ok jake ,will discuss detail about this tomorrow.
Cheers

**Comment by Morningstar88** [comment]
Alright. What time?

6pm?

**Comment by sadrev25** [comment]
Ok let’s see tomorrow

**Comment by Morningstar88** [comment]
ok, see you at 

6pm.
Peace!

**Comment by sadrev25** [comment]
But jake sharmila that one asking 1600 ntd but she will furantee to give more than 50 volunteers who will work js and promotor.I have 1000 ntd ,i will give for urgent.afterwards when it’s possible to give back,take time jake.Is it possible arrange budget 600 ntd tomorrow or i will try to get 600 ntd.
Will see tomorrow

**Comment by Morningstar88** [comment]
Alright, 600TWD is fine, if we are focused on fundraising and javascript volunteers. 

I will write up the plan tomorrow. 
Take care!

**Comment by sadrev25** [comment]
Yeah will see tomorrow

**Comment by sadrev25** [comment]
Hi, jake 7 volunteers contacted me with a referral by the lady, but they are asking 5 USD per each to start our work and after that project, we need to give 10%, I don't know whether to accept this offer or not jake,5 USD ok but is 10% for them ok? what's your opinion?

**Comment by Morningstar88** [comment]
Hi! that might be alright. 5USD is fair...

Let's talk tonight.

430pm confirmed?

Brainstorm idea: we should make videos called "#BuildingKalki"... even with an audience of 10 coders on youtube, we could build it very quickly...

**Comment by Morningstar88** [comment]
430pm tonight confirmed. Are you there?

I have already started programming Kalki!

We will integrate it with the ticker...

**Comment by sadrev25** [comment]
Sure jake

**Comment by Morningstar88** [comment]
hi...

445pm please!!

So busy programming two AIs!!!!

**Comment by sadrev25** [comment]
Yeah jake

**Comment by sadrev25** [comment]
Then I had one emergency ‼️ ,will talk about this today meeting jake!
See you then!

**Comment by sadrev25** [comment]
See you at 4 45 in temple
Cheers

**Comment by Morningstar88** [comment]
Good morning! 

I am free about 9am.

Are you?
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 19, 2023, 16:09 by ***@***.***:

**Comment by Morningstar88** [comment]
Good morning!

Are you sleeping? hahaha.
we can meet 2 hours after you message me...

**Comment by sadrev25** [comment]
Sorry jake,I slept now only i woke up

**Comment by Morningstar88** [comment]
haha.. never mind. let me know when you are free next. today, tomorrow, or after new year... 
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 20, 2023, 11:27 by ***@***.***:

**Comment by sadrev25** [comment]
Is possible tonight will meet i have one meeting upto 7 pm after that we can meet,Is ok for you?

**Comment by sadrev25** [comment]
Hi can we meet today? At 7 pm

**Comment by Morningstar88** [comment]
Hi! Sorry, Im very tired now...

Can meet maybe, but Im very tired,. have zero money...

what would we do?

**Comment by sadrev25** [comment]
Ok jake !will see on another day because you tired so we couldn’t go to secret door

**Comment by sadrev25** [comment]
Will try to meet today jake ,if you are free.During chinese new year everything will close .will have a schedule next days to mediate.

**Comment by Morningstar88** [comment]
no problem! take care...

**Comment by Morningstar88** [comment]
Hello, I have some good news...

Are you free tomorrow at 430pm?

Maybe we have a fun meeting...

**Comment by Morningstar88** [comment]
Hi! Please try to come.... a friend might get us a job, as a team, today !?!?!?!?!?!?!!!!!

¯\_(ツ)_/¯

If nothing happens, you can go party in Xiangshan, but i think it'll be a good chance.

430pm in the temple if you're free...

**Comment by Morningstar88** [comment]
hi, are you coming?

**Comment by sadrev25** [comment]
I told you right jake, came ro elephant mountain with my friends

**Comment by sadrev25** [comment]
> Hi! Please try to come.... a friend might get us a job, as a team, today !?!?!?!?!?!?!!!!! ¯\_(ツ)_/¯ If nothing happens, you can go party in Xiangshan, but i think it'll be a good chance. 430pm in the temple if you're free...
Any good news jake ,we can meet tomorrow?

**Comment by Morningstar88** [comment]
Yes, you told me, but I was going to meet someone interesting, at a door. You also said you would maybe meet, so I just let you know... anyway. nvm...

I am fresh and happy today. We can meet and talk about AIs, and doors.

Will get Kalki onine very soon, so we can go RAISE MONEY!

**Comment by sadrev25** [comment]
Shall we morning itself?jake

**Comment by sadrev25** [comment]
9am ok let me know

**Comment by Morningstar88** [comment]
Hi!

What time?

**Comment by Morningstar88** [comment]
What shall we do? 
i can't meet til at least ten... i need to shower, etc...

**Comment by sadrev25** [comment]
Ok jake will meet in evening

**Comment by sadrev25** [comment]
> What shall we do?  i can't meet til at least ten... i need to shower, etc...

We can feel peace and mediate

**Comment by Morningstar88** [comment]
Cool, tell me by 1pm. Peace!

**Comment by Morningstar88** [comment]
Alright, good idea...

Lets talk later... also, next meeting, please bring your laptop too, if you want... Kalki is LIVE!!!

https://github.com/Morningstar88/X/blob/main/README.md

**Comment by sadrev25** [comment]
Iam going to hsinchu ,to visit temple for Chinese new year jake,will see another day
Cheers

**Comment by Morningstar88** [comment]
sure, another day is better for me too!

have a great day...

hsinchu is fun.
-- 
Ask yourself this question: 'Will this matter a year from now?'



Jan 22, 2023, 13:07 by ***@***.***:

**Comment by sadrev25** [comment]
Hi jake , how its chinese new year is going?

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>Hello Sir!

Having crazy fun with AIs... I built about ten...!! Even one in Chinese we can show to Lion's club.

Will try and make a Tamil one today...

May I ask... who is your favourite Tamil God, or Warrior...

Please name one...

Let me know if you're free.

Best Wishes,

Jake

**Comment by sadrev25** [comment]
My favourite god is ganesha,you can make ai character for him

**Comment by Morningstar88** [comment]
Aha, will do that!

PS, Did your stipend come yet?

**Comment by sadrev25** [comment]
No jake due to chinese new year delayed ,I don’t  know why they closing everything during this days

**Comment by sadrev25** [comment]
Can We start to give assignments to volunteers??? Its still holding

**Comment by sadrev25** [comment]
Everything is still hold.we need to be hurry and make decisions to improve

**Comment by Morningstar88** [comment]
Hi,

1) Please ask them to visit our issues page and introduce themselves, then I can give them a task. 

We can fundraise, the AIs are ready... next Monday everything will open again.

The banks may open today or tomorrow.

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>Hi Mukesh!

Are you still there?

Or did you go home?

Don't worry about the money...

We can continue working here, or when you are back.

I have 35 AIs built now!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

They are listed on my Repo...

beta.character.ai is truly amazing

**Comment by Morningstar88** [comment]
Message ID: ***@***.***>Alright, that's cool..

Let me know when you are rested, and want to raise money online...

We can start fundraising from India.
